web/src/main/webapp/WEB-INF/log4j.cfg

```diff
@@ -62,14 +62,12 @@ log4j.logger.org.jzkit=WARN, console, jeeves
 log4j.logger.org.jzkit.a2j=WARN, console, jeeves
 log4j.logger.org.jzkit.search.impl.LRUCache = WARN, console,jeeves
 
-### JEEVES APPENDER ############################################################
+### JEEVES APPENDER - AODN #####################################################
 
-log4j.appender.jeeves            =org.apache.log4j.DailyRollingFileAppender
-log4j.appender.jeeves.DatePattern='.'yyyy-MM-dd
-log4j.appender.jeeves.file       = logs/geonetwork.log
-#log4j.appender.jeeves.file = ${catalina.home}/logs/geonetwork.log
-log4j.appender.jeeves.layout     = org.apache.log4j.PatternLayout
-log4j.appender.jeeves.layout.ConversionPattern=%d{ISO8601} %-5p [%c] - %m%n
+log4j.appender.jeeves               =org.apache.log4j.RollingFileAppender
+log4j.appender.jeeves.file          = logs/geonetwork.log
+log4j.appender.jeeves.MaxFileSize   =10MB
```

Comment:
Looks to me like that will produce 8 or 9 files per day for catalogue-imos - so we'd keep only about 3-4 days worth of logs.